lms/product/product.py

```diff
@@ -47,8 +47,12 @@ class Settings:
     groups_enabled: bool = False
     """Is the course groups feature enabled"""
 
+    create_lineitem: bool = False
+    """Create lineitem on assignment configuration"""
+
     def __post_init__(self, product_settings):
         self.groups_enabled = product_settings.get("groups_enabled", False)
+        self.create_lineitem = product_settings.get("create_lineitem", False)
```

Comment:
Are we doing this here because it's the only place to do it? This might suggest a flaw in the design, because we will accumulate every possible setting for every possible product no matter which plugin it relates to here.